core/src/main/java/org/kabeja/entities/Polyline.java

```diff
@@ -730,4 +731,22 @@ public void setDocument(DraftDocument doc) {
     public void transform(TransformContext context) {
       
     }
+
+    @Override
+    public void toWcs() {
+        Extrusion e = this.getExtrusion();
+        boolean hasDefaultExtrusion = e.compareToNormalVector(0,0,1);
+        if (hasDefaultExtrusion || is3D()) {
+            return;
+        }
+
+        for (Vertex v : vertices) {
+            v.transformToWcs(this.getExtrusion());
+        }
+    }
+
+    private boolean is3D() {
+        //8: 3d polyline; 16: 3d polygon mesh
+        return ((flags & 8) == 8) || ((flags & 16) == 16);
```

Comment:
@JohannesMessner You could add these as class variables so you don't need to add the comment :)